### DISCUSSION TITLE
Lazy load in riverpod

### DISCUSSION THREAD

**Discussion by billbull21** [rrousselGit/riverpod#1322] [Q&A]
does any body want to tell me how can i do lazy load using this state management?
i have spent my whole day to find out, but i got nothing.

**Comment by rrousselGit** [comment]
What's the problem exactly?

All providers are lazy loaded by default

**Reply by billbull21** [reply]
sorry, i mean,, infinite scroll.

**Reply by billbull21** [reply]
cuz, i thought... riverpod had different way to do than provider or getx.

**Reply by rrousselGit** [reply]
You can look at the marvel example

**Reply by billbull21** [reply]
would you give me some of the code, please? at least only part that i contain lazy load using riverpod.

**Reply by rrousselGit** [reply]
https://github.com/rrousselGit/river_pod/blob/b8ba3e019c0d887a5bf1d8612ebaf0dcc393302d/examples/marvel/lib/src/screens/home.dart#L90

**Reply by billbull21** [reply]
thanks man.

**Reply by billbull21** [reply]
hard to understand :D
i decided to back using provider :(
